docs/by-example/client.rst

```diff
@@ -139,6 +139,24 @@ Refresh locations:
 
     >>> cursor.execute("REFRESH TABLE locations")
 
+Updating Data
+=============
+
+Both when inserting or updating data, values for ``TIMESTAMP`` columns can be obtained
+in different formats. Both scalar strings and datetime objects are supported.
```

Comment:
sorry, not familiar, what is `scalar string`? (is there one of the examples below?

Reply:
I was just not familiar myself with this term, I'm used to `literals` to describe them, but really as you prefer.

Reply:
It's a good suggestion. I rephrased the statement.

> Both literal strings and datetime objects are supported.